openapi/openapi.json

```
@@ -161,6 +161,145 @@
         }
       } ]
     },
+    "/brokers/{brokerCode}/ibans" : {
+      "get" : {
+        "description" : "Internal | External | Synchronous | Authorization | Authentication | TPS | Idempotency | Stateless | Read/Write Intense | Cacheable\n-|-|-|-|-|-|-|-|-|-\nY | N | Y | ApiKey | ApiKey | 1.0/sec | Y | Y | Read | Y\nReturn the list of Ibans of a Broker",
```

```suggestion
        "description" : "Internal | External | Synchronous | Authorization | Authentication | TPS | Idempotency | Stateless | Read/Write Intense | Cacheable\n-|-|-|-|-|-|-|-|-|-\nN | Y | Y | ApiKey | ApiKey | 1.0/sec | Y | Y | Read | Y\nReturn the list of Ibans of a Broker",
```

can you fix this error in the other APIs too? Thanks